site/components/pages/Resources/index.tsx

```diff
@@ -9,14 +9,23 @@ import { Navigation } from "components/Navigation";
 import { PageHead } from "components/PageHead";
 import { Card } from "components/Card";
 import { PodcastCard } from "components/PodcastCard";
+import { ArticlesSlider } from "./FeaturedArticles/ArticlesSlider";
 
-import { Document } from "lib/queries";
+import { Document, FeaturedPost } from "lib/queries";
 
 export interface Props {
   documents: Document[];
+  featuredArticles: FeaturedPost[];
 }
 
-export const Resources: NextPage<Props> = ({ documents }) => {
+export const Resources: NextPage<Props> = ({ documents, featuredArticles }) => {
+  console.log("WORKS", featuredArticles);
```

Comment:
loggies

Reply:
Yes these were there for you 😄 

Removed now as "real" featured articles are queried now